src/components/scenes/CreateWalletSelectFiatScene.tsx

```diff
@@ -98,6 +101,21 @@ const CreateWalletSelectFiatComponent = (props: Props) => {
     navigation.navigate('createWalletCompletion', { createWalletList, walletNames, fiatCode: fiat.value })
   })
 
+  const handleSplit = useHandler(async () => {
+    if (splitSourceWalletId != null) {
+      newWalletItems.forEach(async item => {
+        try {
+          const splitWalletId = await account.splitWalletInfo(splitSourceWalletId, account.currencyConfig[item.pluginId]?.currencyInfo.walletType)
+          const splitWallet = await account.waitForCurrencyWallet(splitWalletId)
+          await splitWallet.renameWallet(walletNames[item.key])
+        } catch (error: any) {
```

Comment:
As a general rule, we prefer `error: unknown` over `error: any`, so TypeScript will yell at us if we try to open up the error object without type-checking it first.

The `showError` function accepts `unknown`, so there is no downside to the extra type-safety.